app/src/main/res/values-es/strings.xml

```diff
@@ -235,64 +275,74 @@
   <string name="listallchannels_share_all">Compartir todos los canales…</string>
   <string name="listallchannels_share">Compartir</string>
   <string name="listallchannels_funding_tx">Transacción</string>
-  <string name="listallchannels_funding_tx_desc">Ver transacción de pagos</string>
+  <string name="listallchannels_funding_tx_desc">Ver transacción de financiación</string>
```

Comment:
What do you think about `Ver transacción de apertura del canal` or even the shortest one : `Ver transacción de apertura`?

`Transacción de financiamiento` could also work just fine and keep the relevance of`funding`

https://www.diccionariodedudas.com/financiacion-o-financiamiento/

Reply:
"Ver transacción **de apertura del canal**" is perfect, thanks.